test/functional/RasapiTest/build.xml

```diff
@@ -0,0 +1,111 @@
+	<property name="dist.dir" location="./dist" />
+	<property name="src" location="." />
+	<property name="LIB" value="junit4"/>
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
+	<!-- <property name="build.cp" value="${BUILD_ROOT}/build/j9jcl/source/ive/lib/jclSC18/classes-vm.zip"/> -->
+	
+	<path id="build.cp">
+		<fileset  dir="${LIB_DIR}/" includes="junit4.jar" />
+	</path>
+	
+	<target name="help">
+		<echo>Ant build script to generate com.ibm.jvm.ras.tests.jar
+		</echo>
+	</target>
+	
+	<target name="clean">
+		<echo>Cleaning build directory ${build.dir}</echo>
+		<mkdir dir="${dist.dir}"/>
+		<delete failonerror="false" verbose="true">
+			<fileset dir="${dist.dir}" includes="${build.jar.tests}" />
+		</delete>
+		<mkdir dir="${build.dir}/bin"/>
+		<delete failonerror="false" verbose="false">
+			<fileset dir="${build.dir}" includes="bin/**/*" />
+		</delete>
+	</target>
+	
+	<target name="compile" depends="clean">
+
+		<echo>Ant version is ${ant.version}</echo>
+		<echo>Compiling code to ${build.dir} and packaging ${build.jar.tests} to ${dist.dir}</echo>
+				
+		<echo>============COMPILER SETTINGS============</echo>
+		<echo>===fork:				yes</echo>
+		<echo>===executable:			${compiler.javac}</echo>
+		<echo>===source:				${src.level}</echo>
+		<echo>===target:				${target.level}</echo>
+		<echo>===debug:					on</echo>
+		<echo>===destdir:				${build.dir}</echo>
+		
+		<mkdir dir="${build.dir}/bin"/>
+		<mkdir dir="${dist.dir}"/>
+		<javac destdir="${build.dir}/bin" source="${src.level}" target="${target.level}" 
+			debug="true" fork="true" executable="${compiler.javac}" encoding="UTF-8" classpathref="build.cp">
+			<!-- Locally there can be a problem with old versions of the DTFJ API which don't support Image.close().
+			 	 I could remove that call but it causes us to leak files and blocks checking if files are system
+			 	 dumps so this points to a recent version of dtfj-interface.jar to compile against instead. -->
+			<!-- <compilerarg value="-Xbootclasspath/p:${build.cp}"/> -->
+			<src path="${source.dir}"/>
+			<!-- The boot compiler, unless it's IBM 1.8, won't have the right API to build all these tests.
+				 We have a hollowed out version in dummy_src for the tests to compile against -->
+			<src path="${dummy.source.dir}"/>
+		</javac>
+		
+		<!-- jar the .class files except the dummy source -->
+		<jar destfile="${dist.dir}/${build.jar.tests}">
+			<fileset dir="${build.dir}/bin">
+				<exclude name="com/ibm/jvm/*.class" />
+			</fileset>
+		</jar>
+
+    	<copy todir="${dist.dir}">
+    		<fileset  dir="${LIB_DIR}/" includes="junit4.jar" />
+    	</copy>
+		
+    	<copy todir="${dist.dir}">
+    		<fileset dir="." includes="test.xml"/>
+    	</copy>
+	</target>
+	
+    <target name="build" depends="compile">
+    	<mkdir dir="${DEST}"/>
+		<copy todir="${DEST}">
+			<fileset dir="${dist.dir}" includes="*.xml,**/*.jar,**/*.xml"/>
+			<fileset dir="${src}" includes="*.mk"/>
+		</copy>
+    </target>
```

Comment:
This test is for openj9/ibm, so we should disable the compilation on the other jvm as well.